clients/zbctl/cmd/activate.go

```diff
@@ -0,0 +1,29 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// activateCmd represents the activate command
+var activateCmd = &cobra.Command{
+	Use:   "activate",
+	Short: "Activate a resource",
```

Comment:
Activate an job not a resource right?

Reply:
zbctl has currently a structure like `zbctl ACTION RESOURCE`. That's why we often write resource at the action level as there could be more. For example create instance and job.